MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvxAppCompatViewPresenter.cs

```diff
@@ -0,0 +1,269 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Android.Support.Design.Widget;
+using Android.Support.V4.App;
+using Android.Support.V4.Util;
+using Android.Support.V4.View;
+using Android.Support.V7.App;
+using MvvmCross.Core.ViewModels;
+using MvvmCross.Core.Views;
+using MvvmCross.Droid.Support.V4;
+using MvvmCross.Droid.Views;
+using MvvmCross.Droid.Views.Attributes;
+using MvvmCross.Platform;
+using MvvmCross.Platform.Core;
+using MvvmCross.Platform.Exceptions;
+using MvvmCross.Platform.Platform;
+
+namespace MvvmCross.Droid.Support.V7.AppCompat
+{
+    public class MvxAppCompatViewPresenter : MvxAndroidViewPresenter
+    {
+        public MvxAppCompatViewPresenter(IEnumerable<Assembly> androidViewAssemblies) : base(androidViewAssemblies)
+        {
+        }
+
+        protected new FragmentManager CurrentFragmentManager 
+        {
+            get
+            {
+                if(CurrentActivity is AppCompatActivity activity)
```

Comment:
This should check for `FragmentActivity` since `SupportFragmentManager` hangs off that.